SmartDeviceLink/SDLH264VideoEncoder.m

```diff
@@ -159,6 +159,11 @@ - (BOOL)encodeFrame:(CVImageBufferRef)imageBuffer presentationTimestamp:(CMTime)
 
     OSStatus status = VTCompressionSessionEncodeFrame(_compressionSession, imageBuffer, presentationTimestamp, kCMTimeInvalid, NULL, (__bridge void *)self, NULL);
 
+    if (status == kVTInvalidSessionErr) {
```

Comment:
Add a comment here to explain why this code was added since it is a hack to fix a Video Toolbox bug. Something like:
> // HAX: On some older iPhones `VTCompressionSessionEncodeFrame` fails with a `kVTInvalidSessionErr` as soon as the device is locked. Attempt to fix this by recreating the compression session.